tensorflow/core/distributed_runtime/master.h

```diff
@@ -94,6 +94,9 @@ class Master {
   // Cleanup unused session.
   void GC();
 
+  // Find master session.
+  MasterSession* FindMasterSession(const string& handle);
```

Comment:
This method increments the refcount on the returned `MasterSession*`, so the comment should indicate that.